reactions/src/SpecialFunctions.cpp

```diff
@@ -1101,16 +1102,18 @@ int funcn2(double a, const double G[], double F[], void *params)
 	real p1 = p.par1;
   real p2 = p.par2;
 	real p3 = p.par3;
+	int  p4 = p.par4;
+	real omeganu = p.par5;
 
+	double omegacb = omega0-omeganu;
 
-	double hade1 = HA2g(a,omega0,p1,p2,p3);
-	double hade2 = HA2g2(a,omega0,p1,p2,p3);
+	double hub1 = pow2(HAg(a,omega0,p1,p2,p3));
+	double hub2 = HA1g(a,omega0,p1,p2,p3);
+	double ap5 = pow(a,5);
 
+	F[0] = G[1];
```

Comment:
form of new ODE is equivalent to old form but changed for testing sensitivity - also needed to change initial conditions